src/systems/filecoin_vm/actor/address/address.go

```diff
@@ -4,6 +4,16 @@ package address
 // 	util "github.com/filecoin-project/specs/util"
 // )
 
+// Addresses for singleton system actors
+var (
+	InitActorAddr           = &Address_I{} // TODO
+	CronActorAddr           = &Address_I{} // TODO
+	StoragePowerActorAddr   = &Address_I{} // TODO
+	StorageMarketActorAddr  = &Address_I{} // TODO
+	PaymentChannelActorAddr = &Address_I{} // TODO
+	BurntFundsActorAddr     = &Address_I{} // TODO
+)
```

Comment:
this stuff shouldn't leak into this package. `address` is fundamental -- it shouldn't know about any of these actors.